lib/active_admin/base_controller.rb

```diff
@@ -22,18 +22,18 @@ class << self
       attr_accessor :active_admin_config
     end
 
+    include Menu
+    include Authorization
```

Comment:
See 7ec7d2f3f  My guess is this is an accidental (untested?) regression caused by a rebase.

Reply:
@varyonic  I changed the order. But I am sure that the order of the includes can't affect the code that is in the `navigation_menu` block, since the block and related methods are never executed while `BaseController` is loaded. It is only definition at that moment.